setup.py

```diff
@@ -172,7 +172,13 @@ def run(self):
     license='LGPLv2.1+',
     platforms=['any'],
     py_modules=['libusb1'],
-    packages=['usb1'],
+    packages=['usb1', 'usb1.__pyinstaller'],
```

Comment:
Maybe setuptools.findpackages can be used in the future here?

Reply:
It should indeed. I think this hardcoding is a leftover from when python-libusb1 was just 2 modules without a package. I will switch to it after merging, thanks for the heads-up.